WeScan/Review/ReviewViewController.swift

```diff
@@ -9,11 +9,13 @@
 import UIKit
 
 /// The `ReviewViewController` offers an interface to review the image after it has been cropped and deskwed according to the passed in quadrilateral.
-final class ReviewViewController: UIViewController {
+open class ReviewViewController: UIViewController {
```

Comment:
I'm not a fan of making all the view controllers open. We are working on a possible addition of preferences that can be passed in to WeScan - is that not a better way of handling this? I feel like they were made final for a reason - let me know what you think.

Reply:
I agree with you here, @justJS. I wonder what the reason is to make the view controllers open. If it's styling, we can also do this through adding styling possibilities with injection